src/shared/lib/IGVUtils.ts

```diff
@@ -3,6 +3,7 @@ import {CopyNumberSeg} from "shared/api/generated/CBioPortalAPI";
 import {TrackProps} from "shared/components/igv/IntegrativeGenomicsViewer";
 
 export const WHOLE_GENOME = "all";
+export const DEFAULT_GENOME = "hg19";
```

Comment:
if the reference genome is a page wide setting, I don't think this should go in IGVUtils. It should go in ResultsViewPageStoreUtils